simple/classes/model.xml

```diff
@@ -0,0 +1,8 @@
+<yo:object xmlns:yo="http://framework.yuapan.com/ood" module="simple" lang="en">
+    <class name="model" userName="Model" id="9c2728d5-2c60-4bc1-bce4-748317c86ef6" shortDescription="title"
+           longDescription="title description">
+        <slot name="title" type="YTstring" userName="Title" mandatory="true"/>
+        <slot name="description" type="YTstring" userName="Description"/>
+        <slot name="content" type="YTstring" userName="Content" computed="true" getter="cGetContent"/>
+    </class>
+</yo:object>
```

Comment:
I decided to leave YTMoney and YTRefs behind the brackets for now and make something simpler to test full cycle. When we have a skeleton we can build up some muscles on top of it.